Docs/developer_guide/building_instructions/windows.md

```diff
@@ -1,2 +1,143 @@
 # Microsoft Windows
 
+## Install prerequisites
+
+- [CMake](http://www.cmake.org/cmake/resources/software.html) >= 3.15.1
+- [Git](https://git-scm.com/download/win) >= 1.7.10
+  - Note CMake must be able to find `git.exe` and `patch.exe`. If git is installed in the default location then they may be found there, but if they are not found then either add the folder that contains them to `PATH` environment variable; or set `GIT_EXECUTABLE` and `Patch_EXECUTABLE` as environment variables or as CMake variables at configure time.
+- [NSIS](http://nsis.sourceforge.net/Download) (optional): Needed if packaging Slicer. Make sure you install the language packs.
+- [Qt5](https://www.qt.io/download-open-source): Download Qt universal installer and install Qt 5.15 along with qtscript and qtwebengine components.
+- [Visual Studio](https://visualstudio.microsoft.com/downloads/)
+  - any edition can be used (including the free Community edition)
+  - when configuring the installer, enable installation of Visual Studio 2019 (v142) toolset and component Programming languages / Visual C++ / Common Tools for Visual C++ (in some distributions, this option is not enabled by default)
+
+Other versions:
+- Visual Studio 2017 (v141) toolset is not tested anymore but probably still works
```

Comment:
@lassoan Based on a discussion over at https://github.com/Slicer/Slicer/issues/4958#issuecomment-669886361, if you're using a binary dependency that is built with MSVC2019 (v142), such as Qt 5.15, you will need to compile Slicer with MSVC 2019 (v142) as well.

Reply:
Thanks for the note. Probably Slicer can be still built using VS2017 but because Qt-5.15 needs VS2019 redistributables, users may need to install Visual C++ redistributable package (or VS2019 redistributables have to be included in the install package).

Reply:
Yeah not sure if it needs to be mentioned in the instructions that if you are going to build with v141 and using the Qt 5.15 binary with v142, that you are going to need those latest v142 redistributables.  They are already included in the installed package if you build Slicer with v142.